i_scene_cp77_gltf/meshtools/meshtools.py

```diff
@@ -246,14 +246,14 @@ def applyRefitter(obj):
             gskey.value = 0
             gskey = setActiveShapeKey(obj, name)
             bpy.ops.object.shape_key_remove(all=False)
-    newnames = getShapeKeyNames(obj)
     bpy.context.view_layer.objects.active = obj
 
-    # if we have active shape keys: activate 'Basis' and remove the others
-    if obj.data.shape_keys is not None and obj.data.shape_keys.key_blocks is not None:
-        setActiveShapeKey(obj, 'Basis')
-        bpy.ops.object.shape_key_remove(all=False)
+    # if we have active shape keys: activate 'Basis' and remove it
+    if not setActiveShapeKey(obj, 'Basis'):
+        raise Exception("Failed to activate 'Basis' shape key")
```

Comment:
Please use BaseException (or whatever it was called, search `raise new` or so), because Exception can't actually be caught